ts/test/test-utils/stubs/ciphers/FallBackSessionCipherStub.ts

```diff
@@ -0,0 +1,11 @@
+import { CipherTextObject } from '../../../../window/types/libsignal-protocol';
+import { SignalService } from '../../../../protobuf';
+
+export class FallBackSessionCipherStub {
+  public async encrypt(buffer: ArrayBuffer): Promise<CipherTextObject> {
```

Comment:
why don't make this class and `SecretSessionCipherStub` full static? tslint complains about it but might be good to be able to just use `FallBackSessionCipherStub.encrypt()` rather than having to create a temporary instance.
What do you think?

Reply:
We can't because the original class `FallBackSessionCipher` is not static. All this stub is emulating is that class so it works in a test environment.